### PR TITLE
[PT FE]: support aten::gelu with tanh approximate and tests

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *supported aten::gelu with tanh approximate*

### CODE REVIEW

**Review by mvafin — CHANGES_REQUESTED** [review]
Please do `ConvertLike` to `x` type for all constants here, because otherwise you translator will only work for fp32 `x`

**Comment by mvafin** [comment]
NVM, please use `Gelu(x, op::GeluApproximationMode::TANH);`

**Comment by eaidova** [comment]
> NVM, please use `Gelu(x, op::GeluApproximationMode::TANH);`

moved on this approach

File: src/frontends/pytorch/src/op/gelu.cpp [hunk 1]
```diff
@@ -16,9 +24,35 @@ OutputVector translate_gelu(NodeContext& context) {
...
+        auto beta = context.mark_node(std::make_shared<ov::op::v1::Divide>(two, pi));
+        auto beta_sqrt = context.mark_node(std::make_shared<ov::op::v0::Sqrt>(beta));
+        three = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(three, x));
+        auto x3 = context.mark_node(std::make_shared<ov::op::v1::Power>(x, three));
+        kappa = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(kappa, x));
+        auto kappa_x3 = context.mark_node(std::make_shared<ov::op::v1::Multiply>(kappa, x3));
+        auto x_kappa_x3 = context.mark_node(std::make_shared<ov::op::v1::Add>(x, kappa_x3));
+        beta_sqrt = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(beta_sqrt, x));
+        auto inner_prod = context.mark_node(std::make_shared<ov::op::v1::Multiply>(beta_sqrt, x_kappa_x3));
+        auto inner_tanh = context.mark_node(std::make_shared<ov::op::v0::Tanh>(inner_prod));
```
[the remark this answers is not in the document]
Comment (eaidova) [reply src/frontends/pytorch/src/op/gelu.cpp]: no need anymore

**Review by mvafin — APPROVED** [review]

**Pull request merged**